frontend/app/src/views/source-data/SourceDataUpload.vue

```diff
@@ -215,7 +193,7 @@ export default {
           formData.append('fileExtractedDate', this.fileExtractedDate);
           // formData.append('processStartDate', '2024-01-15T21:45:15.842Z');
           // formData.append('processEndDate', '2024-01-15T21:45:15.842Z');
-          formData.append('userId', 'PTUGGER');
+          formData.append('userId', this.userName);
```

Comment:
are you waiting for me on this to stop passing this?

Reply:
Until now we had it hardcoded, So I am just fetching the logged-in user and passing the dynamic userId for now, To your question, yes Once you implemented getting this ID from the KC token just ignore this param from the request Body.